### DISCUSSION TITLE
Add selection interaction to remove/clean measures dialogs?

### DISCUSSION THREAD

**Discussion by Sy256** [helge17/tuxguitar#128] [General]
The way it works with copy measure should also be on remove/clean measures, where it starts with the selected measures in the dialog box. Thanks!

**Comment by guiv42** [comment]
done (dbd1321a09f9817be931c1bfa361dc67f4b52390)
Will be available in next pre-release

**Reply by Sy256** [reply]
Ok thanks! Have you been able to look into this bug? It keep happening more often (mostly on solo mode) and it's getting annoying to have to go back a measure or two just to experiment on one note https://github.com/helge17/tuxguitar/issues/77

**Reply by guiv42** [reply]
remove/clean measures dialog update is now available in last pre-release.
You don't need to send reminders about #77. You have done the right thing creating the issue. OK, no one answered. It doesn't mean that nobody cares, but just that no one *could* find any solution.

**Reply by Sy256** [reply]
Ok thank you! Just wasn't sure if you/anyone saw that bug yet. Topic closed
